build/pkgs/gmp/spkg-install.in

```diff
@@ -28,37 +28,6 @@ fi
 
 
 case "$UNAME" in
-    SunOS)
-        true;;  # Auto-detect ABI
-    Darwin)
-        # In some cases (see SAGE_ROOT/spkg/bin/sage-env), on Darwin,
```

Comment:
Perhaps we may keep `Darwin);;` as a stub?

Reply:
I don't think this would be an improvement. Doing nothing is the default when no case matches

Reply:
OK.